linetools/spectra/xspectrum1d.py

```diff
@@ -766,13 +767,19 @@ def rebin(self, new_wv, all=False, **kwargs):
         all : bool, optional
           Rebin all spectra in the XSpectrum1D object?
           Set masking='none' to have the resultant spectra all be regsitered, but note
-             that there will still be masking
+             that there will still be masking unless otherwise specified in kwargs
```

Comment:
I did not understand the comment regarding setting masking to 'none'. What is the idea here?

Also, check typo `regsitered`.